src/bridgewatcher/discord/util/md.py

```
@@ -0,0 +1,24 @@
+class Markdown:
+    def italic(self, s: Any) -> str:
+        return f"*{str(s)}*"
+
+    def bold(self, s: Any) -> str:
+        return f"**{str(s)}**"
+
+    def bold_italic(self, s: Any) -> str:
+        return f"***{str(s)}***"
+
+    def codeblock(self, s: Any) -> str:
+        return f"`{str(s)}`"
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)